src/scripts/update.ts

```diff
@@ -20,6 +22,23 @@ const fetchPreviousTokens = async (): Promise<Token[]> => {
   }
 };
 
+/**
+ * Fetches the previously generated icons.
+ *
+ * @return {Promise<Icon[]>} Array of previously generated icons
+ */
+const fetchPreviousIcons = async (): Promise<Icon[]> => {
+  logger.info("Fetching previouly generated icons...");
+  try {
+    const { icons } = await import(`../.${LIB_TOKENS}`);
+    logger.info(`Found ${icons.length} previously generated icons.`);
+    return [...icons] as Icon[];
```

Comment:
on réellement besoin de ce cast ?

Reply:
du log ou de la diff ?

Reply:
J'ai pas compris aha je parlais du cast, ce serait cool d'avoir:
```suggestion
    return [...icons];
```